src/macad_gym/carla/config.json

```diff
@@ -25,7 +25,7 @@
             "reward_function": "corl2017",
             "manual_control": false,
             "auto_control": false,
-            "camera_type": "rgb",
+            "camera_type": "rgb-0",
```

Comment:
yeah, this is more of a proposal since i wanted to make also the position customizable: the number (if present) allow to select which camera position between the default existing, to use. Do you think that the compliance is more important respect the feature or it is not necessary?

Reply:
I see.  For identifying and specifying the camera or it's position, a camera ID (or position ID) can be used. 
The `camera-type` should be consistent with CARLA's camera type.

Reply:
do you suggest to add a new parameter e.g. `camera-position` ?

Reply:
Yes, to add support for multiple cameras mounted on the car with configurable positions, adding `camera_position` sounds better than changing the `camera_type`.

Reply:
Ok, I reviewed also the other comments but I will close them as soon as I will be able to fix them in the code